vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -5,20 +5,20 @@ sidebarDepth: 2
 
 ## Overview
 
-The Entando Command Line Interface (CLI) provides a set of commands that accelerate common tasks such as installing a new copy of Entando, generating an Entando project via JHipster, deploying an Entando bundle, etc.
+The Entando Command Line Interface (CLI) provides a set of commands that accelerate common tasks such as installing a new copy of Entando, generating an Entando project via JHipster, deploying an Entando Bundle, etc.
```

Comment:
not sure if "version" covers all use cases; let's go with "instance"

common works here; it's the usual language to describe tasks/actions that are prevalent. i think we need it because it emphasizes that the cli is helpful wrt routine operations and doesn't just address rare/fringe use cases